### PR TITLE
Clean-up code

### PR DESCRIPTION
**Pull request by allanlewis**
I wanted to clean-up the code before making any substantive changes.

### CODE REVIEW

**Comment by allanlewis** [comment]
I've also raised this upstream: liris/websocket-client#249

**Comment by allanlewis** [comment]
I've rebased to squash a fix for a (completely legitimate) test failure.

**Comment by allanlewis** [comment]
@johnteslade, it would be great if you could review this at some point. Even if it's rejected upstream, it's the basis for future work, so I'd like to merge into our fork.

File: websocket/_abnf.py [hunk 1]
```diff
@@ -389,6 +412,7 @@ def extract(self, frame):
         self.cont_data = None
         frame.data = data[1]
         if not self.fire_cont_frame and data[0] == ABNF.OPCODE_TEXT and not self.skip_utf8_validation and not validate_utf8(frame.data):
```
Comment (johnteslade) [inline websocket/_abnf.py:415]: Split this line to 80 width?

File: bin/wsdump.py [hunk 2]
```diff
@@ -63,7 +68,9 @@ def parse_args():
 
     return parser.parse_args()
 
-class RawInput():
+
+class RawInput:
```
Comment (johnteslade) [inline bin/wsdump.py:72]: Should this be a new style class `RawInput(object)`?

Comment (allanlewis) [inline bin/wsdump.py:72]: Yeah, it should be, but I wasn't sure if that would have some adverse impact downstream, and the test coverage isn't very high. If you don't think there would be any adverse impact, I'll change it.

File: websocket/_abnf.py (hunk 1 above)
Comment (allanlewis) [inline websocket/_abnf.py:415]: Good spot: fixed in 01b88d3.

**Pull request closed without merging**